docs/learn-transactions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: learn-transactions
+title: Transactions
+sidebar_label: Transactions
+---
+
+Transactions are used to send a balance to another user. To begin sending transactions, we will
+learn how to using Polkadot User Interface.
+
+## Polkadot UI
```

Comment:
I usually call it Polkadot-JS Apps as its proper name, although Polkadot UI seems friendlier.